packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -282,18 +277,38 @@
   font-weight: bold;
 }
 
-.rdp-day_selected:not([aria-disabled='true']),
-.rdp-day_selected:focus:not([aria-disabled='true']),
-.rdp-day_selected:active:not([aria-disabled='true']),
-.rdp-day_selected:hover:not([aria-disabled='true']) {
+.rdp-day_selected:not(:disabled),
+.rdp-day_selected:focus:not(:disabled),
+.rdp-day_selected:active:not(:disabled),
+.rdp-day_selected:hover:not(:disabled) {
   background-color: var(--rdp-accent-color);
   color: white;
 }
 
-.rdp-day_selected:focus:not([aria-disabled='true']) {
+.rdp-day_selected:focus:not(:disabled) {
   border: var(--rdp-outline-selected);
 }
 
+.rdp-day_disabled {
+  overflow: visible;
+  color: var(--color-muted);
+  pointer-events: none;
+
+  &::after {
+    background: var(--color-muted);
+    content: '';
+    display: block;
+    position: absolute;
+    width: calc(100% + 4px);
+    height: 2px;
+
+    @media (forced-colors: active) {
+      background-color: GrayText;
+      cursor: not-allowed;
```

Comment:
Oh, that's interesting! Good catch. When I go back and look at [their PR that switched back from aria-disabled to disabled](https://github.com/gpbl/react-day-picker/pull/1476/files#diff-92f59a8fb79262d38fdd7b3331ed126b223acea9ca037292dbeda4a9e0a62dbdL60), it looks like they removed the `pointer-events: none` because it's not needed when the browser already knows the button is `disabled`.

Reply:
Fixed in [a107aa5](https://github.com/CMSgov/design-system/pull/3064/commits/a107aa577f868fc5a9462800a92a4f3ba7807e57)